docs/fr/README.md

```diff
@@ -0,0 +1,354 @@
+
+- [Introduction](#docs-introduction)
+- [Traductions](#docs-translations)
+- [Configuration](#docs-configuration)
+- [Utilisations Simples](#docs-ideas)
+- [Usage](#docs-usage)
+- [Helpers](#docs-helpers)
+- [Usage Courant De QrCode](#docs-common-usage)
+- [Usage Hors De Laravel](#docs-outside-laravel)
+
+<a id="docs-introduction"></a>
+## Introduction
+Simple QrCode est un adaptateur facile d'utilisation pour le framework Laravel et qui s'appuie sur le magnifique travail fourni par [Bacon/BaconQrCode](https://github.com/Bacon/BaconQrCode). Nous avons conçu une interface intuitive, facile d'installation et familière aux utilisateurs de Laravel.
+
+<a id="docs-translations"></a>
+## Traductions
+Nous recherchons des utilisateurs parlant arabe, espagnol, français, coréen ou japonnais pour nous aider à traduire cette documentation. Si vous vous en sentez capable, créez une pull request ! 
+
+<a id="docs-configuration"></a>
+## Configuration
+
+#### Composer
+
+Commencez par ajouter le paquet QrCode à la section `require` de votre fichier `composer.json`: 
+
+	"require": {
+		"simplesoftwareio/simple-qrcode": "~1"
+	}
+
+Lancez ensuite la commande `composer update`.
+
+#### Service Provider
+
+###### Laravel 4
+Ajouter l'entrée `SimpleSoftwareIO\QrCode\QrCodeServiceProvider` au tableau `providers` du fichier de configuration `app/config/app.php`.
+
+###### Laravel 5
+Ajouter l'entrée `SimpleSoftwareIO\QrCode\QrCodeServiceProvider::class` au tableau `providers` du fichier de configuration `config/app.php`.
+
+#### Alias
+
+###### Laravel 4
+Enfin, ajoutez l'entrée `'QrCode' => 'SimpleSoftwareIO\QrCode\Facades\QrCode'` au tableau `aliases` du fichier de configuration `app/config/app.php`.
+
+###### Laravel 5
+Enfin, ajoutez l'entrée `'QrCode' => SimpleSoftwareIO\QrCode\Facades\QrCode::class` au tableau `aliases` du fichier de configuration `config/app.php`.
+
+<a id="docs-ideas"></a>
+## Utilisations Simples
+
+#### Print View
+
+L'un des pricipaux usages que nous faisons de ce paquet et d'avoir des QrCodes dans toutes nos vues d'impression. Cela donne la possibilité à nos clients qui le flashent de revenir à la page d'origine du document après que celui-ci a été imprimé. Nous obtenons cette fonctionnalité en ajoutant le code suivant au fichier footer.blade.php.
+
+	<div class="visible-print text-center">
+		{!! QrCode::size(100)->generate(Request::url()); !!}
+		<p>Flashez-moi pour revenir à la page d'origine.</p>
+	</div>
+
+#### Embarquer Un QrCode
+
+Vous pouvez embarquer un qrcode dans un courriel pour permettre à vos utilisateurs de le flasher. Voici un exemple pour mettre ceci en œuvre dans Laravel :
+
+	// Dans un template blade.
+	<img src="{!!$message->embedData(QrCode::format('png')->generate('Embarquez-moi dans un courriel!'), 'QrCode.png', 'image/png')!!}">
+
+<a id="docs-usage"></a>
+## Usage
```

Comment:
« Utilisation » and « Usage » could both be used. I prefer « Usage » in this case because it has a technical connotation.